content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -333,8 +333,10 @@ support [Network Policy](/docs/concepts/services-networking/network-policies/).
 See a list of add-ons that implement the
 [Kubernetes networking model](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-network-model).
 
-You can install a Pod network add-on with the following command on the
-control-plane node or a node that has the kubeconfig credentials:
+
+Please refer to the Installing Addons page for a non-exhaustive list of networking 
+addons supported by Kubernetes You can install a Pod network add-on with the following 
+command on the control-plane node or a node that has the kubeconfig credentials:
```

Comment:
The "Installing Addons" text seems to be missing the inline link to the page, and there's also a minor punctuation oversight in the new line.

```suggestion
Please refer to the [Installing Addons](/docs/concepts/cluster-administration/addons/#networking-and-network-policy) page for a non-exhaustive list of networking 
addons supported by Kubernetes. You can install a Pod network add-on with the following 
command on the control-plane node or a node that has the kubeconfig credentials:
```

Reply:
Correction added as part tof latest commit. Thanks!